src/array.jl

```diff
@@ -729,6 +729,7 @@ function in(x::CategoricalValue, y::CategoricalArray{T, N, R}) where {T, N, R}
     end
 end
 
+Array(x::CategoricalArray{T}) where {T} = convert(Array{T}, x)
 collect(x::CategoricalArray{T}) where {T} = convert(Array{T}, x)
```

Comment:
This could also be written as `Array(x::CategoricalArray{T}) where {T} = Array{T}(x)`. Not sure which is preferred.

Reply:
I think in theory `Array{T}(x)` should be preferred since constructors (and `collect` BTW) are supposed to make copies, but `convert` doesn't have to. In practice since we're dealing with a `CategoricalArray` input it shouldn't make a difference since there's no way to transform it into an `Array` without making a copy.

Reply:
Ok, I changed to Array{T}(x) to make the copy more explicit. Also changed the variable name from `x` to `A` to follow naming conventions.